openmetadata-service/src/test/java/org/openmetadata/service/OpenMetadataApplicationTest.java

```diff
@@ -54,6 +54,7 @@ public abstract class OpenMetadataApplicationTest {
   public static DropwizardAppExtension<OpenMetadataApplicationConfig> APP;
   protected static final WebhookCallbackResource webhookCallbackResource = new WebhookCallbackResource();
   public static final String FERNET_KEY_1 = "ihZpp5gmmDvVsgoOG6OVivKWwC9vd5JQ";
+  public static Jdbi jdbi;
```

Comment:
To init the MigrationWorkflow in the tests we need the `jdbi`. LMK if there's a better way to pass this without changing the var here